test/parallel_api/ranges/move_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,69 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+
+#include "support/pstl_test_config.h"
+
+#if _ENABLE_RANGES_TESTING
+#include <oneapi/dpl/ranges>
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ENABLE_RANGES_TESTING
+    constexpr int max_n = 10;
+    int data[max_n] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+    int data2[max_n];
+
+    auto lambda1 = [](auto i) { return i * i; };
+
+    using namespace oneapi::dpl::experimental::ranges;
```

Comment:
I would be careful with `using namespace oneapi::dpl::experimental::ranges` and then making unqualified call. We faced with problems with that approach previously due to ADL. If you want to make calls shorter I would recommend to use `using namespace oneapi::dpl::experimental;` and then call `ranges::move` here and in all other tests.

Reply:
In general, I agree.
But, here it just a test...  a very trivial test. There are no potential problems here.
Even, if we decide to use ranges:: for each name in the tests, we have to do it everywhere in the tests  - 29 files;
We do really need to do it now?

Reply:
Discussed offline. It can be done latter for the all tests